Gems/Atom/RHI/Vulkan/Code/Source/RHI/NullDescriptorManager.cpp

```diff
@@ -211,15 +211,19 @@ namespace AZ
                 imageCreateInfo.arrayLayers = m_imageNullDescriptor.m_images[imageIndex].m_arrayLayers;
                 imageCreateInfo.flags = m_imageNullDescriptor.m_images[imageIndex].m_imageCreateFlagBits;
 
-                VkResult result = vkCreateImage(device.GetNativeDevice(), &imageCreateInfo, nullptr, &m_imageNullDescriptor.m_images[imageIndex].m_image);
+                VkResult result = device.GetContext().CreateImage(
+                    device.GetNativeDevice(), &imageCreateInfo, nullptr, &m_imageNullDescriptor.m_images[imageIndex].m_image);
                 RETURN_RESULT_IF_UNSUCCESSFUL(ConvertResult(result));
 
                 VkMemoryRequirements memReqs = {};
-                vkGetImageMemoryRequirements(device.GetNativeDevice(), m_imageNullDescriptor.m_images[imageIndex].m_image, &memReqs);
+                device.GetContext().GetImageMemoryRequirements(
+                    device.GetNativeDevice(), m_imageNullDescriptor.m_images[imageIndex].m_image, &memReqs);
 
                 // image device memory
                 m_imageNullDescriptor.m_images[imageIndex].m_deviceMemory = device.AllocateMemory(memReqs.size, memReqs.memoryTypeBits, VK_MEMORY_PROPERTY_DEVICE_LOCAL_BIT);
-                result = vkBindImageMemory(device.GetNativeDevice(), m_imageNullDescriptor.m_images[imageIndex].m_image, m_imageNullDescriptor.m_images[imageIndex].m_deviceMemory->GetNativeDeviceMemory(), 0);
+                result = device.GetContext().BindImageMemory(
+                    device.GetNativeDevice(), m_imageNullDescriptor.m_images[imageIndex].m_image,
```

Comment:
Something went wrong with the indentation here, similarly below

Reply:
I'm using clang format and this line is longer than the maximum allowed line length - that's why it changed. I already started a discussion in discord about more consistent clang-format use. The conclusion so far was that a slow transition toward a formatted codebase is preferred, i.e. formatting parts that are being changed instead of formatting the whole code base at once. See #dev-languages. I'd love to see a more official policy regarding this though. ;)